soh/src/overlays/gamestates/ovl_file_choose/z_file_choose.c

```diff
@@ -268,7 +903,12 @@ void FileChoose_FinishFadeIn(GameState* thisx) {
 void SpriteLoad(FileChooseContext* this, Sprite* sprite) {
     OPEN_DISPS(this->state.gfxCtx);
 
-    if (sprite->im_siz == G_IM_SIZ_16b) {
+    if (sprite->im_siz == G_IM_SIZ_8b) {
+        gDPLoadTextureBlock(POLY_OPA_DISP++, sprite->tex, sprite->im_fmt,
+                            G_IM_SIZ_8b, // @TEMP until I figure out how to use sprite->im_siz
```

Comment:
I'm not sure if it was understood at the time, but this comment doesn't explain the problem/current workaround. Let's update this comment and the ones below into one comment above as something along the lines of:

> Due to macro expansion and the token-pasting operator (##), we cannot pass `sprite->im_siz` in directly. Instead we must call `gDPLoadTextureBlock` with the raw IM_SIZ define name itself to properly expand the correct defines internally.